src/telegram.js

```diff
@@ -845,6 +845,165 @@ class TelegramBot extends EventEmitter {
+   * @returns {Promise}
+   * @see https://core.telegram.org/bots/api#exportchatinvitelink
+   */
+  exportChatInviteLink(chatId, form = {}) {
+    form.chat_id = chatId;
+    return this._request('exportChatInviteLink', { form });
+  }
+
+  /**
+   * Use this method to set a new profile photo for the chat. Photos can't be changed for private chats.
+   * The bot must be an administrator in the chat for this to work and must have the appropriate admin rights.
+   * Returns True on success.
+   *
+   * @param  {Number|String} chatId  Unique identifier for the message recipient
+   * @param  {String|stream.Stream|Buffer} photo A file path or a Stream. Can
+   * also be a `file_id` previously uploaded
+   * @param  {Object} [options] Additional Telegram query options
+   * @return {Promise}
+   * @see https://core.telegram.org/bots/api#setchatphoto
+   */
+  setChatPhoto(chatId, photo, options = {}) {
+    const opts = {
+      qs: options,
+    };
+    opts.qs.chat_id = chatId;
+    try {
+      const sendData = this._formatSendData('photo', photo);
+      opts.formData = sendData[0];
+      opts.qs.photo = sendData[1];
+    } catch (ex) {
+      return Promise.reject(ex);
+    }
+    return this._request('setChatPhoto', opts);
+  }
+
+  /**
+   *
+   * @param {Number|String} chatId  Unique identifier for the message recipient
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
+   * @see https://core.telegram.org/bots/api#deletechatphoto
+   */
+  deleteChatPhoto(chatId, form = {}) {
+    form.chat_id = chatId;
+    return this._request('deleteChatPhoto', { form });
+  }
+
+  /**
+   * Use this method to change the title of a chat. Titles can't be changed for private chats.
+   * The bot must be an administrator in the chat for this to work and must have the appropriate admin rights.
+   * Returns True on success.
+   *
+   * Note: In regular groups (non-supergroups), this method will only work if the ‘All Members Are Admins’
+   * setting is off in the target group.
+   *
+   * @param {Number|String} chatId  Unique identifier for the message recipient
+   * @param {String} title New chat title, 1-255 characters
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
+   * @see https://core.telegram.org/bots/api#setchattitle
+   */
+  setChatTitle(chatId, title, form = {}) {
+    form.chat_id = chatId;
+    form.title = title;
+    return this._request('setChatTitle', { form })
+  }
+
+  /**
+   * Use this method to change the description of a supergroup or a channel.
+   * The bot must be an administrator in the chat for this to work and must have the appropriate admin rights.
+   * Returns True on success.
+   *
+   * @param {Number|String} chatId  Unique identifier for the message recipient
+   * @param {String} description New chat title, 1-255 characters
+   * @param {Object} [options] Additional Telegram query options
```

Comment:
Since description is a required parameter according to the method described above we can either omit the additional options or only make chatId as the only required option. I think the latter suits it better.